### PR TITLE
Create ecs-fargate.md

### PR DESCRIPTION
**Pull request by yotamloe**
Hey, I created this doc on How to send traces and container metrics from ecs fargate cluster to logz.io.

https://deploy-preview-2329--logz-docs.netlify.app/shipping/tracing-sources/ecs-fargate.html

### CODE REVIEW

File: _source/logzio_collections/_tracing-sources/ecs-fargate.md [hunk 1]
```diff
@@ -0,0 +1,138 @@
...
+The CloudFormation template requires the following parameters:
+
+*   `ClusterName`: The name of your ECS cluster from which you want to collect metrics.
+*   `CreateIAMRoles`: Specifies whether to create default IAM roles (True or False).
+*   `TaskRoleArn`: The role ARN you want to use as the ECS task role. (Optional, only required if you set `CreateIAMRoles` to False)
+*   `ExecutionRoleArn`: The role ARN you want to use as the ECS execution role. (Optional, only required if you set `CreateIAMRoles` to False)
+*   `SecurityGroups`: The list of SecurityGroupIds in your Virtual Private Cloud (VPC).
+*   `Subnets`: The list of Subnets in your Virtual Private Cloud (VPC).
+*   `LogzioTracingToken`: Your Logz.io tracing account token.
+*   `LogzioMetricsToken`: Your Logz.io metrics account token.
```
Comment (mirii1994) [inline _source/logzio_collections/_tracing-sources/ecs-fargate.md:74]: What happens if someone wants to only ship metrics or only ship traces?

File: _source/logzio_collections/_tracing-sources/ecs-fargate.md [hunk 2]
```diff
@@ -0,0 +1,138 @@
...
+*   `LogzioRegion`: Your Logz.io region. for example: `us`
+*   `LogzioListenerUrl`: Your Logz.io listener URL. for example: `https://listener.logz.io:8053`
+
+### Resources and Configuration
```
Comment (mirii1994) [inline _source/logzio_collections/_tracing-sources/ecs-fargate.md]: Can we move the Resources and Configuration section after the steps? We need the instructions part to be as short as possible

**Review by mirii1994 — CHANGES_REQUESTED** [review]
Great work! please see my comments

**Review by Simplychee — APPROVED** [review]

File: _source/logzio_collections/_tracing-sources/ecs-fargate.md (hunk 2 above)
Comment (yotamloe) [reply _source/logzio_collections/_tracing-sources/ecs-fargate.md]: @Simplychee Please :)

File: _source/logzio_collections/_tracing-sources/ecs-fargate.md (hunk 1 above)
Comment (yotamloe) [reply _source/logzio_collections/_tracing-sources/ecs-fargate.md:74]: They would need to change the applied configuration manually

Comment (mirii1994) [reply _source/logzio_collections/_tracing-sources/ecs-fargate.md:74]: So maybe it should be mentioned that somewhere

**Pull request merged**